src/exml.erl

```diff
@@ -112,10 +104,8 @@ to_iolist([_ | _] = Elements, Pretty) ->
     Head = hd(Elements),
     [Last | RevChildren] = lists:reverse(tl(Elements)),
     case {Head, Last} of
-        {#xmlstreamstart{name = Name, attrs = Attrs},
-         #xmlstreamend{name = Name}} ->
-            Element = #xmlel{name = Name, attrs = Attrs,
-                             children = lists:reverse(RevChildren)},
+        {#xmlstreamstart{name = Name, attrs = Attrs}, #xmlstreamend{name = Name}} ->
+            Element = #xmlel{name = Name, attrs = Attrs, children = lists:reverse(RevChildren)},
```

Comment:
A line that's wrapped by hand for record field readability is unwrapped automatically by the formatter.
Expected: One field per line should be preserved.

Reply:
As I stated in another comment, the formatter doesn't care about how you formatted your code "by hand". It just reads AST and prints Erlang code. It will never _preserve_ your style.

Reply:
@elbrujohalcon how about making a rule (possibly configurable) telling the formatter to have one attribute of a record per line, starting with the first attribute in the same line as the record name? Sth like below:
```erlang
            Element = #xmlel{name = Name, 
                             attrs = Attrs,
                             children = lists:reverse(RevChildren)},
```

Would that make sense?

Reply:
Yeah… that's exactly the intention of `inline_items`. That's why it's weird that it's not working here.